### ISSUE TITLE
sql2csv config file not handling delimiter mapping correctly

### ISSUE THREAD

**Issue by rsdoiel** [caltechlibrary/datatools#14]
Looks like a have a bug in the 1.2.0 release. If you set the delimiter to tab it still comes out comment when using the JSON configuration.  

JSON example

```
{
    "dsn_url": "sqlite://file:mydb.sqlite3",
    "delimiter": "\t"
}
```

Should result in a tab delimited output when run a query is run via sql2csv.  I wonder if the slash escaping isn't happening correctly when I convert the string which represents a single character to the rune for the CSV writer in the Comma attribute.

**Comment by rsdoiel** [comment]
Work around is to use the -delimiter command line option.

**Comment by rsdoiel** [comment]
Fixed in the upcoming 1.2.1 release.
